Tests/SwiftSyntaxBuilderTest/StringInterpolationTests.swift

```diff
@@ -408,6 +408,26 @@ final class StringInterpolationTests: XCTestCase {
     )
   }
 
+  func testCodeBlockItemInterpolation() {
+    let codeBlockItem: CodeBlockItemSyntax =
+      """
+      func foo() {
+        return bar
+      }
+      """
+
+    XCTAssertTrue(codeBlockItem.is(CodeBlockItemSyntax.self))
+    AssertStringsEqualWithDiff(
```

Comment:
This function has been converted to lowercase. Could you update it and make sure that tests build and pass locally?

Reply:
Ohhh...I selected ``swift-parser-cli`` as the scheme and it just skipped ``SwiftSyntaxBuilderTest``. Sorry for that.